core/src/main/java/org/axonframework/deadline/quartz/DeadlineJob.java

```diff
@@ -101,7 +101,7 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
             }
         } catch (Exception e) {
             LOGGER.error("Exception occurred during processing a deadline job [{}]", jobDetail.getDescription(), e);
-            throw new JobExecutionException(e);
+            throw new JobExecutionException(e, true);
```

Comment:
I would prefer you'd use a named constant instead of putting `true` in there. 
So what I am suggesting is adding a `private static final Boolean REFIRE_IMMEDIATELY = true`, which you'd use here i.o. just `true`.

Reply:
It's done